protobufs/generated/messages/events.proto

```diff
@@ -71,26 +71,689 @@ enum EventType {
 
   CHART_OPENED = 60;
 
-  // Command events
```

Comment:
Would these generic ones never fire? I wonder if we should keep and allow usage to determine whether the specific or generic events are appropriate for their needs.

Reply:
I figured that it's better to be specific than generic. Do you see a case where someone would want a protocol to act on any type of command?

Reply:
I agree that most plugins will be better served by specific events, but I don't think we should limit plugin authors in this manner. Consider this plugin for a customer to measure their own command originations.

```python
from canvas_sdk import Handler
from canvas_sdk.events import Event
from canvas_sdk.utils import post_request, utcnow_iso_8601

class CommandOriginationMetrics(Handler):
    RESPOND_TO = [POST_COMMAND_ORIGINATE]

    def compute(event: Event):
        post_request("http://metrics.example.com/increment-command-origination-count",
            {
                "event_id": str(event.id),
                "command": event.context['schema_key'],
                "event_timestamp": str(utcnow_iso_8601()),
            }
        )
```

Reply:
That's a good use case. I'll add them back.